src/client/app/components/maps/MapCalibrationInfoDisplayComponent.tsx

```diff
@@ -88,7 +88,8 @@ export default class MapCalibrationInfoDisplayComponent extends React.Component<
 		const longitudeIndex = 1;
 		if (this.props.currentCartesianDisplay === 'x: undefined, y: undefined') { return; }
 		const input = this.state.value;
-		if (isValidGPSInput(input)) {
+		const {validGps} = isValidGPSInput(input);
+		if (validGps) {
```

Comment:
This isn't your doing but would it work if you had an else clause that showed the error if the GPS is invalid? Up to now the code simply does nothing.